packages/imtr/src/config.ts

```diff
@@ -9,9 +9,10 @@ export const apis: APIConfig[] = [
       "dakraam-plaatsen": ["hMwHKR7Wz4FP8Dm4x", "dRy4PfDs7jQPc9gMG"],
       "kozijnen-plaatsen": ["Xm2WwYeGkNN9w6rgQ", "5CJrbgbWZP6uZsouY"],
       "bouwwerk-slopen": [
-        "bxBCdnrFZSbwxgmxC",
-        "2eYY7McaiRgfnERo9",
-        "D4mccynMtNdzYGhXF",
+        // demolition
+        "bxBCdnrFZSbwxgmxC", // monument
+        "D4mccynMtNdzYGhXF", // report
+        "2eYY7McaiRgfnERo9", // bsdg
```

Comment:
Make sure we sort accordingly.

**Warning** ⚠️ 
**We should test all checkers again! To see if ordering still makes sense.**

Reply:
bsdg is a dutch abbreviation. I think it is only clear for insiders.

Reply:
thanks, fixed